src/pages/ANL-payment.html

```diff
@@ -21,15 +21,15 @@ <h3><a href="http://localhost:44349/Results/Link/@Model.ShareIdentifier">View yo
       <callout class="secondary">
         <row>
           <columns large="6">
-            <p>
-                <strong>Online Order Number</strong><br />@Model.Id<br /><br />
-                <strong>Project Title/Location</strong><br/>@order.Project<br /><br />
-                <strong>Payment Method</strong><br />@order.Payment.ClientType<br /><br />
-                @if(order.ClientId != null)
-                {
-                <strong>Client Id</strong><br />@order.ClientId
-                }
-            </p>
+              <p>
+                  <strong>Online Order Number</strong><br />@Model.Id<br /><br />
+                  <strong>Project Title/Location</strong><br />@order.Project<br /><br />
+                  <strong>Payment Method</strong><br />@order.Payment.ClientType<br /><br />
+                  @if(Model.ClientId != null)

```

Comment:
@jSylvestre I thought clientID changed locations?

Reply:
@srkirkland Model is Order.ClientId. order is OrderDetails.